dev_apps/nfc-api-test/js/nfc_tech.js

```diff
@@ -36,7 +39,10 @@ var NfcTechDemo = {
 
     if (tag.techList.indexOf('ISO-DEP') != -1) {
       this.processIsoDep(tag);
-      sucMsg.textContent = 'transceive succeeds.';
+      sucMsg.textContent = 'transceive ISO-DEP succeeds.';
+    } else if (tag.techList.indexOf('NFC-A') != -1 && tag.type) {
```

Comment:
Thanks.
Current nfc implementation, for a particularly tag, we could read tech type as NFC-A but tag.type = null. It would throw error in this case